ui/server/src/main/scala/pl/touk/nussknacker/ui/definition/defaults/ConfigParameterDefaultValueDeterminer.scala

```diff
@@ -0,0 +1,19 @@
+package pl.touk.nussknacker.ui.definition.defaults
+
+import pl.touk.nussknacker.engine.api.defaults.{NodeDefinition, ParameterDefaultValueDeterminer}
+import pl.touk.nussknacker.engine.api.definition.Parameter
+import pl.touk.nussknacker.engine.api.process.ParameterConfig
+
+class ConfigParameterDefaultValueDeterminer(config: ParamDefaultValueConfig) extends ParameterDefaultValueDeterminer {
```

Comment:
Now we have Determiner and Extractor for very similar classes.

ParameterConfigValidatorsExtractor / ParameterConfigEditorExtractor vs ConfigParameterDefaultValueDeterminer.

Maybe we should equalize all mentioned names?

Reply:
Indeed WDYT about using Determiner in all cases? Extraction is special case of determining when logic is very straightforward. Should I do it in this change or another one?

Reply:
i would go for "now" option :)

Reply:
Ok I'll do it